matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/session/room/RoomService.kt

```diff
@@ -97,6 +97,12 @@ interface RoomService {
      */
     fun getRoomSummary(roomIdOrAlias: String): RoomSummary?
 
+    /**
+     * A live [RoomSummary] associated with the room with id [roomId].
+     * You can observe this summary to get dynamic data from this room, even if the room is not joined yet
+     */
+    fun getRoomSummaryLive(roomId: String): LiveData<Optional<RoomSummary>>
```

Comment:
It feels a bit weird that this uses `Optional` and the non-live version uses a nullable. Is it possible to use the same for both?

Reply:
`LiveData` cannot emit null value IIRC.